modules/installation-azure-arm-tested-machine-types.adoc

```diff
@@ -0,0 +1,20 @@
+
+// Module included in the following assemblies:
+//
+// installing/installing_azure/installing-azure-customizations.adoc
+// installing/installing_azure/installing-azure-government-region.adoc
+// installing/installing_azure/installing-azure-network-customizations.adoc
+// installing/installing_azure/installing-azure-private.adoc
+// installing/installing_azure/installing-azure-user-infra.adoc
+// installing/installing_azure/installing-azure-vnet.adoc
+
+[id="installation-azure-arm-tested-machine-types_{context}"]
+= Tested instance types for Azure ARM
+
+The following Microsoft Azure instance types have been tested with {product-title}.
+
+.Machine types
+[%collapsible]
+====
+//Will add arm azure instance types when PR#6331 is merged
```

Comment:
@Prashanth684 Hello Prashanth! Other than adding the `md` file here, does everything else in this PR look good?

Reply:
lgtm from me thanks @kelbrown20